src/containters/Home/index.js

```diff
@@ -14,13 +13,14 @@ export const Home = () => {
         // other API 0.7
         // TOTAL :
         Promise.all([heroView()]).then(res => {
+            setHeroViewData(res[0].data);
```

Comment:
- Please remove un-needed comments and Also use console log to log error in case of error until we handle errors gracefully.